src/content/docs/new-relic-solutions/get-started/intro-new-relic.mdx

```diff
@@ -31,41 +31,17 @@ import apmPrometheus from 'images/apm_logo_prometheus.png'
 
-With New Relic, you can:
-
-* **[Bring all your data together](#bring-your-data)**: Instrument everything and import data from across your technology stack using our agents, integrations, and APIs, and access it from a single UI.
-* [**Analyze your data**](#analyze-your-data): Get all your data at your fingertips to find the root causes of problems and optimize your systems. Build dashboards and charts or use our powerful query language.
-* **[Respond to incidents quickly](#respond-faster)**: Our machine learning solution proactively detects and explains anomalies and warns you before they become problems.
-
-<img
-  title="2020-04-28-17.26.02.gif"
-  alt="New Relic - Kubernetes Cluster Explorer"
-  src={kubernetesNewRelicKubernetesClusterExplorer}
+<Video
+  id="nbM9dyRF1Kc"
+  type="youtube"
 />
 
-<figcaption>
-  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Kubernetes cluster explorer**: Our Kubernetes UI is just one of our many features.
-</figcaption>
-
 ## Get started with New Relic [#get-started-now]
 
-Here's how you can quickly get started:
-
-1. If you don't have a New Relic account, sign up at [newrelic.com/signup](https://newrelic.com/signup). It's free, forever!
-2. For a recommended path on how to set up New Relic, see [our quick launch guide](/docs/new-relic-solutions/get-started/quick-launch-guide). If you'd prefer to add monitoring for some specific entities and services, use the [Add your data](https://one.newrelic.com/launcher/nr1-core.settings?pane=eyJuZXJkbGV0SWQiOiJ0dWNzb24ucGxnLWluc3RydW1lbnQtZXZlcnl0aGluZyJ9) UI page. 
-3. Once you have data coming into New Relic, learn more about the [New Relic UI](/docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one) or [set up alerts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/introduction-alerts/).
-
-Want to estimate costs? See [our blog post on estimating costs](https://newrelic.com/blog/nerdlog/estimate-data-cost). 
-
-## All the answers in one place [#answers-in-one-place]
-
-New Relic is built for full stack observability. It links all relevant data so that you get the whole picture of everything that enables your systems to deliver value to your customers, from the container running a microservice in the cloud to a mobile website's shopping cart button.
-
-## Bring all your data together [#bring-your-data]
-
-Capture, organize, and make sense of your data in New Relic, no matter where it comes from. Use our [agents and integrations](/docs/infrastructure/host-integrations/get-started/introduction-host-integrations/) to automatically collect data from common frameworks and tools, or use [our APIs](/docs/apis/get-started/intro-apis/introduction-new-relic-apis) for data that's more specific to your business or technology.
+1. [Sign up for a free account](https://newrelic.com/signup) if you haven't already.
```

Comment:
Maybe instead of 'use our agents' we can be more specific about what we need them to do here. We don't necessarily want them to use the agents _yet_, we want them to install an agent (with any necessary integrations) to get them instrumented... I think "Choose your deployment below to install one of our agents. Don't see your deployment here? Learn more about how to instrument your system with [some of our available integrations](the link here)."

@bradleycamacho